embedded-kafka/pom.xml

```diff
@@ -17,7 +17,7 @@
         <camel.version>2.18.3</camel.version>
         <camel-kafka.version>2.16.5</camel-kafka.version>
         <io.netty.version>3.7.0.Final</io.netty.version>
-        <kafka_2.11.version>0.8.2.2</kafka_2.11.version>
```

Comment:
why did you touch kafka client version?

Reply:
only Kafka > 0.10.2.0 supports different advertised listeners (https://issues.apache.org/jira/browse/KAFKA-4565).
Different listeners are needed for connecting from another container in scope of installed network and from host machine.

Reply:
ok, but why you touched client?